src/models/resultsSerializer.ts

```diff
@@ -256,16 +280,26 @@ export default class ResultsSerializer {
     /**
      * Open the saved file in a new vscode editor pane
      */
-    public openSavedFile(filePath: string): void {
+    public openSavedFile(filePath: string, format: string): void {
         const self = this;
-        let uri = vscode.Uri.file(filePath);
-        self._vscodeWrapper.openTextDocument(uri).then((doc: vscode.TextDocument) => {
-            // Show open document and set focus
-            self._vscodeWrapper.showTextDocument(doc, 1, false).then(undefined, (error: any) => {
+        if (format === 'excel') {
+            // This will not open in VSCode as it's treated as binary. Use the native file opener instead
+            // Note: must use filePath here, URI does not open correctly
+            opener(filePath, undefined, (error, stdout, stderr) => {
```

Comment:
>opener(filePath [](start = 12, length = 15)

did you have a chance to see what this does on mac, linux, and windows w/o excel installed?  As long as it doesn't blow-up the behavior should be fine.